<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simulation</string>
  </property>
  <property name="windowIcon">
   <iconset resource="quitIcon.qrc">
    <normaloff>:/img/plane.png</normaloff>:/img/plane.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>600</width>
      <height>600</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>1</number>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="backgroundBrush">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>0</red>
       <green>170</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="foregroundBrush">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>60</y>
      <width>371</width>
      <height>571</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>331</width>
       <height>551</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButtonStart">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Start</string>
          </property>
          <property name="icon">
           <iconset resource="quitIcon.qrc">
            <normaloff>:/img/play.png</normaloff>:/img/play.png</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButtonStop">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Stop</string>
          </property>
          <property name="icon">
           <iconset resource="quitIcon.qrc">
            <normaloff>:/img/stop.png</normaloff>:/img/stop.png</iconset>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButtonRestart">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Restart</string>
          </property>
          <property name="icon">
           <iconset resource="quitIcon.qrc">
            <normaloff>:/img/restart.png</normaloff>:/img/restart.png</iconset>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButtonReset">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Reset</string>
          </property>
          <property name="icon">
           <iconset resource="quitIcon.qrc">
            <normaloff>:/img/reset.png</normaloff>:/img/reset.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="labelCurrentFile">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>60</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>150</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="text">
           <string>Speed</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0" rowspan="3">
           <widget class="QSlider" name="verticalSliderSpeed">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>150</height>
             </size>
            </property>
            <property name="mouseTracking">
             <bool>false</bool>
            </property>
            <property name="tabletTracking">
             <bool>false</bool>
            </property>
            <property name="toolTipDuration">
             <number>1</number>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>61</number>
            </property>
            <property name="pageStep">
             <number>13</number>
            </property>
            <property name="value">
             <number>30</number>
            </property>
            <property name="sliderPosition">
             <number>30</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksBothSides</enum>
            </property>
            <property name="tickInterval">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>x60</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>x2</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLCDNumber" name="lcdNumberSpeed">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>60</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="text">
           <string>Time</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumberTimeMin">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="smallDecimalPoint">
           <bool>true</bool>
          </property>
          <property name="digitCount">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="text">
           <string>min</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumberTimeSec">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="smallDecimalPoint">
           <bool>true</bool>
          </property>
          <property name="digitCount">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="text">
           <string>sec</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>30</y>
      <width>311</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Control panel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1051</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionReset"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuScenario">
    <property name="title">
     <string>Scenario</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionGenerate"/>
    <addaction name="actionGenerate_Open"/>
    <addaction name="actionConfigure"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuScenario"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="quitIcon.qrc">
     <normaloff>:/img/remove.png</normaloff>:/img/remove.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionGenerate">
   <property name="icon">
    <iconset resource="quitIcon.qrc">
     <normaloff>:/img/magic-wand.png</normaloff>:/img/magic-wand.png</iconset>
   </property>
   <property name="text">
    <string>Generate</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="icon">
    <iconset resource="quitIcon.qrc">
     <normaloff>:/img/settings.png</normaloff>:/img/settings.png</iconset>
   </property>
   <property name="text">
    <string>Configure</string>
   </property>
  </action>
  <action name="actionSaveScreen">
   <property name="icon">
    <iconset resource="quitIcon.qrc">
     <normaloff>:/img/diskette.png</normaloff>:/img/diskette.png</iconset>
   </property>
   <property name="text">
    <string>Save Screen</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="quitIcon.qrc">
     <normaloff>:/img/share.png</normaloff>:/img/share.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset resource="quitIcon.qrc">
     <normaloff>:/img/reset.png</normaloff>:/img/reset.png</iconset>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionGenerate_Open">
   <property name="icon">
    <iconset resource="quitIcon.qrc">
     <normaloff>:/img/magic-wand.png</normaloff>:/img/magic-wand.png</iconset>
   </property>
   <property name="text">
    <string>Generate and Open</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="quitIcon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>494</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
